guides/micronaut-token-propagation/micronaut-token-propagation.adoc

```diff
@@ -220,9 +218,11 @@ Now you can call the `/user` endpoint supplying the access token in the Authoriz
 [source,bash]
 ----
 curl "http://localhost:8080/user" -H 'Authorization: Bearer eyJhbGciOiJIUzI1NiJ9.eyJzdWIiOiJzaGVybG9jayIsIm5iZiI6MTYxNTkxMDM3Nywicm9sZXMiOltdLCJpc3MiOiJnYXRld2F5IiwiZXhwIjoxNjE1OTEzOTc3LCJpYXQiOjE2MTU5MTAzNzd9.nWoaNq9YzRzYKDBvDw_QaiUyVyIoc6rHCW_vLfnrtQ8'
+----
 
+```
```

Comment:
I am not sure that this is proper asciidoc. I think they are Github fenced code blocks